proto/path/protocol/morse.proto

```diff
@@ -10,8 +10,10 @@ enum MorseEndpointErrorType {
   MORSE_ENDPOINT_ERROR_TIMEOUT = 2;
   MORSE_ENDPOINT_ERROR_MAXED_OUT = 3;
   MORSE_ENDPOINT_ERROR_MISCONFIGURED = 4;
-  MORSE_ENDPOINT_ERROR_INVALID_RESPONSE = 5;
-  MORSE_ENDPOINT_ERROR_INTERNAL = 6;  // Added for internal gateway errors
+  MORSE_ENDPOINT_ERROR_TLS_CERTIFICATE_VERIFICATION_FAILED = 5;
+  MORSE_ENDPOINT_ERROR_INVALID_RESPONSE = 6;
+  MORSE_ENDPOINT_ERROR_NON_JSON_RESPONSE = 7;
```

Comment:
#PUC to clarify this error is returned by the SDK, i.e. `morse` package is not trying to unmarshal the endpoint's returned payload into JSON.